src/rinterface_extra.c

```diff
@@ -2382,11 +2382,10 @@ igraph_error_t R_igraph_progress_handler(const char *message, double percent,
   SEXP l3 = PROTECT(Rf_lang2(l1, l2));
   SEXP rho = PROTECT(Rf_eval(l3, R_BaseEnv));
 
-  SEXP l4 = PROTECT(Rf_install(".igraph.progress"));
-  SEXP l5 = PROTECT(Rf_ScalarReal(percent));
-  SEXP l6 = PROTECT(Rf_ScalarString(PROTECT(Rf_mkChar(message))));
-  SEXP l7 = PROTECT(Rf_lang3(l4, l5, l6));
-  PROTECT(ec=Rf_eval(l7, rho));
+  SEXP l4 = PROTECT(Rf_ScalarReal(percent));
```

Comment:
@schochastics if you agree we can remove the tk (not tkplot) stuff, I will need help with this C file. I am getting many errors from the current changes.